war/.eslintignore

```diff
@@ -0,0 +1,20 @@
+work
+target
+/rebel.xml
+
+# Node
+node/
+node_modules/
+
+# Generated JavaScript Bundles
+jsbundles
+
+src/main/webapp/css/font-awesome
+src/main/webapp/css/google-fonts
+src/main/webapp/css/icomoon
+
+# External scripts
+src/main/webapp/scripts/yui
+src/main/webapp/scripts/prototype.js
+src/main/webapp/scripts/*.js
```

Comment:
But `src/main/webapp/scripts/hudson-behavior.js` is an internal script, not an external script.

https://github.com/jenkinsci/jenkins/pull/8012